pkg/config/controller_config.go

```diff
@@ -43,21 +43,66 @@ func GetClusterLocalGateway() (string, error) {
 	return DefaultServiceNetwork, nil
 }
 
-func ConfigInit() {
-	// discover VPC using environment first
-	VpcID = os.Getenv("CLUSTER_VPC_ID")
```

Comment:
Is there any reason to change this?  Today we documented [how to use these environments](https://github.com/aws/aws-application-networking-k8s/blob/main/docs/enviroment.md)

Reply:
This document is for When running AWS Gateway API Controller `outside` the Kubernetes Cluster, this specify the VPC of the cluster. This code changes is to take the input while running AWS Gateway API Controller  `inside` the he Kubernetes Cluster.